ru/turing/courses/lesson4/filippov/NotFoundError.java

```diff
@@ -0,0 +1,19 @@
+package ru.turing.courses.lesson4.filippov;
+
+public class NotFoundError extends RuntimeException {
```

Comment:
Лучше переделать на
```suggestion
public class NotFoundException extends RuntimeException {
```
Error - это все-таки то, что мы не можем отловить